src/arrays/ArrayUtil.java

```diff
@@ -12,21 +14,45 @@
      */
     public static void createArray(int size) {
 
+        int [] newArray= new int[size];
+        for(int i = 0; i < newArray.length; i++) {
+            newArray[i] = i;
+            System.out.print(newArray[i] + " ");
+        }
+        System.out.println();
     }
 
     /**
      * Create two dimensional array with the given sizes, fill it with different values and print it.
      */
     public static void create2dArray(int x, int y) {
+
+        int[][] newArray = new int[x][y];
+        int k = 4;
+        for (int i = 0; i < x; i++) {
+            for (int j = 0; j < y; j++) {
+                newArray[i][j] = k;
+                System.out.print(newArray[i][j] + " ");
+                k += 3;
+            }
+            System.out.println();
+        }
+        System.out.println();
     }
 
+
     /**
      * Add the given value at the beginning of the given array
      *
      * @return new array.
      */
     public static int[] addFirst(int[] array, int valueToAdd) {
 
+        array[0] = valueToAdd;
```

Comment:
The point here is not to change the value of the first element of the array, but to add the value to the beginning of the array. 
You can write like this
   int length = array.length + 1;
        int[] newArray = new int[length];
        for (int i = 0; i < length; i++) {
            if (i == 0) {
                newArray[i] = valueToAdd;
            } else {
                newArray[i] = array[i - 1];
            }
        }
        return newArray;